Svc/ActiveLogger/test/ut/TestMain.cpp

```diff
@@ -0,0 +1,15 @@
+// ----------------------------------------------------------------------
```

Comment:
It looks like this file is still in its stub form.  We should probably remove it.

Reply:
Removed unused file.